app/models/master_file.rb

```diff
@@ -723,12 +727,6 @@ def reloadTechnicalMetadata!
                          'Unknown'
                        end
 
-    self.duration = begin
-      @mediainfo.duration.to_s
-    rescue
-      nil
-    end
-
```

Comment:
Removing this appears to break batch entry. Running through the code, batch entry tries to save the master file during processing which triggers the `:after_save` hook for updating the poster image. Because the duration is not set, this causes an error that prevents the save from occurring. 

Should we leave this in place for now and create a new ticket to handle the batch entry case?

Reply:
I'm not sure.  I'm a bit confused as to why the error is happening and why it is blocking save.  I would have expected that the lack of duration would lead to not generated a thumbnail.  Would you be available to walk me through the trace and we could figure out a best path forward?